content/en/docs/bootstrap/azuredo.md

```diff
@@ -0,0 +1,174 @@
+---
```

Comment:
Maybe rename to `azure-devops.md`? Might make it clearer from a URL perspective (SEO maybe too?).